server/gfmd/user.c

```diff
@@ -84,6 +84,20 @@ grpassign_add_group(struct group_assignment *ga)
 	u->groups.group_prev = ga;
 }
 
+gfarm_error_t
+user_auth_id_moidfy(struct user *u, const char *auth_method,
```

Comment:
there is a typo here (due to my fault).
Please replace user_auth_id_moidfy by user_auth_id_modify.